cmd/infracost/register.go

```diff
@@ -137,7 +154,7 @@ func promptForEmail() (string, error) {
 
 func promptOverwriteAPIKey() (bool, error) {
 	p := promptui.Prompt{
-		Label:     "Would you like to overwrite your existing saved API key",
+		Label:     "Would you like to overwrite your existing saved API key?",
```

Comment:
```suggestion
		Label:     "Would you like to overwrite your existing saved API key",
```

promptui adds a `?` by default.

Reply:
oh didn't know that it will add. Thanks, will change it.